schedule/yast/autoyast/autoyast_mini.yaml

```diff
@@ -6,6 +6,9 @@ description: >
   variables. The variables are specified in Job Group or in Test Suite settings.
 vars:
   AUTOYAST_PREPARE_PROFILE: '1'
+  DESKTOP: textmode
+  AUTOYAST_CONFIRM: '1'
+  INSTALLONLY: '1'
```

Comment:
HDDSIZEGB=30 & HDDMODEL=scsi-hd for the same reason as for autoyast_keyboard_layout